SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -108,7 +108,7 @@ void USpatialActorChannel::Init(UNetConnection* InConnection, int32 ChannelIndex
 }
 #endif
 
-void USpatialActorChannel::DeleteEntityIfAuthoritative()
+void USpatialActorChannel::DeleteEntityIfAuthoritative(const bool bTryTombstone /* = true */)
```

Comment:
As a followup change, we can probably remove the bTryTombstone flag, since we will need to handle deletion on PIE shutdown differently.  The reason being that things that are tombstoned in PIE no longer exist on the server, so they can't be deleted on PIE shutdown through existing codepaths.

Reply:
For now, this change fixes the fact that no startup actors were getting deleted on PIE shutdown.

Reply:
Tomorrow i'll be investigating a solution that will catch the tombstoned stuff as well when shutting down PIE